docs/multicluster/antctl.md

```diff
@@ -1,11 +1,15 @@
 # Antctl Multi-cluster commands
 
 Starting from version 1.6.0, Antrea supports the `antctl mc` commands, which can
-collect information from a leader cluster in a ClusterSet, for troubleshooting
-issues in an Antrea Multi-cluster ClusterSet.
+collect information from a leader cluster in a ClusterSet for troubleshooting
+issues in an Antrea Multi-cluster ClusterSet, create and delete resources in an
+Antrea Multi-cluster ClusterSet, and so on. The command `antctl mc get` is supported
+since Antrea version 1.6.0 and other commands are supported from 1.7.0. These commands
+can run from out-of-cluster only. The antctl will look for your kubeconfig file at
```

Comment:
Is it true? Do you mean they cannot run in antrea-controller or agent Pods?

Why they cannot?

Reply:
When we add the "mc" subcommand we used 

```
	for _, cmd := range cl.rawCommands {
		if (runtime.Mode == runtime.ModeAgent && cmd.supportAgent) ||
			(runtime.Mode == runtime.ModeController && cmd.supportController) ||
			(!runtime.InPod && cmd.commandGroup == mc) {
```

in `antctl/comand_list.go` . So there's no runtime.Mode in mc subcommand like other commands.

Reply:
Ok. We should revise the words. It is not really "out-of-cluster". Maybe just say they cannot run inside the `antrea-controller`, `antrea-agent`, and `antrea-mc-controller` Pods.

Reply:
Sure. I will revise the words.